core/testing.go

```diff
@@ -5,11 +5,14 @@ import (
 	"testing"
 
 	"github.com/tendermint/tendermint/abci/example/kvstore"
+	"github.com/tendermint/tendermint/abci/types"
 	"github.com/tendermint/tendermint/config"
 	"github.com/tendermint/tendermint/node"
 	rpctest "github.com/tendermint/tendermint/rpc/test"
 )
 
+var defaultRetainBlocks int64 = 10
```

Comment:
Should this be const?

Reply:
Ooops. Agree. Changing 
```suggestion
const defaultRetainBlocks int64 = 10
```